apps/playground/src/app/modules/components/tile-module/tile-module.component.html

```diff
@@ -100,26 +92,25 @@ <h2>DsfrTileComponent</h2>
       satisfy the <code>DownloadOptions</code> interface :
       <pre>
 <span class='highlight-code'>export interface DownloadOptions</span> &#123;
-    open?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be downloaded and opened right away</span>
-    openOnAnotherPage?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be opened on another page</span>
-    fileName?: string; <span class='highlight-comment'>// The name of the file when downloaded. Leave empty to keep the name from the server</span>
-    fileLang?: string; <span class='highlight-comment'>// The language of the downloaded file</span>
-    fileDesc?: string; <span class='highlight-comment'>// An optional description of the file</span>
+  open?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be downloaded and opened right away</span>
+  openOnAnotherPage?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be opened on another page</span>
+  fileName?: string; <span class='highlight-comment'>// The name of the file when downloaded. Leave empty to keep the name from the server</span>
+  fileLang?: string; <span class='highlight-comment'>// The language of the downloaded file</span>
+  fileDesc?: string; <span class='highlight-comment'>// An optional description of the file</span>
     fileType: string; <span class='highlight-comment'>// The extension of the file</span>
     fileWeight: string; <span class='highlight-comment'>// The expected weight for the file, ex: '42 Ko'</span>
-&#125;</pre>
+&#125;
+      </pre>
       ⚠️ Please be aware that <code>fileType</code> and
       <code>fileWeight</code> are mandatory.<br />
       By default, the other properties of the interface, if not provided, will
       be set to the values :
-      <pre>
-&#123;
+      <pre>&#123;
```

Comment:
Please check in the interface the indentation for the everything within the `pre` tags : that's what we use to represent code and is the only HTML tag that preserves the indentation and white-space that we give it